roles/multibench_run/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+# If this shell cmd ahs a correct rc, it means the multibench_host is ready
+- name: "Check the multi-bench host has the crucible binaries"
+  ansible.builtin.shell: >
+    crucible help
+  delegate_to: multibench
+  become: true
+
+# This file contains some information about the execution like the number of iterations, the tags...
+- name: "Generate config file for multibench"
+  ansible.builtin.template:
+    src: templates/config.ini.j2
+    dest: "{{ multibench_script_dir }}/config.ini"
+    mode: '0644'
+  delegate_to: multibench
+  become: true
+
+- name: "Creating an output directory if needed"
+  when:
+    - multibench_run_output_dir is undefined
+  block:
+    # If the multibench_output_dir is not defined, it generates a tmp directory to store the results.
+    - name: "Create working directory"
+      ansible.builtin.tempfile:
+        state: directory
+        prefix: multibench
+      register: _multibench_run_output
+
+    - name: "Set the new directory as multibench_run_output_dir"
+      ansible.builtin.set_fact:
+        multibench_run_output_dir: _multibench_run_output
+
+# Multibench needs to be run as root to properly access all settings files needed.
+# The script launch a crucible cmd which start to build the container images used for the tests.
+# Then, the infrastructure is deployed in a dedicated namespace called crucible-rickshaw.
+# Once the tests are finished, the results are published in an OpenSearch DB and a summary is generated.
+- name: "Launch multibench workload"
+  ansible.builtin.shell:
+    cmd: >
+      ./{{ multibench_script }} config.ini
```

Comment:
is there no default value for `multibench_script` here? (which is `run.sh`). Should be placed in the defaults file

Reply:
Good catch! The linter made me do a big refactor and I changed the definition of this variable.

Reply:
ack